src/style.scss

```diff
@@ -14,6 +14,46 @@
   display: inline-block;
 }
 
+:host([fixed]) {
```

Comment:
There's a bunch of other properties set to rems as well (e.g. height, width, row size, padding). If we want the pane to display correctly we need to set those as well.

Reply:
Fixed styles are now automatically generated.